the-collab-lab/tcl-15-smart-shopping-list
# Issue #15 | Shopping List Screen

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles:  Style   |
|  ✓ | 💡  New feature   |

## Acceptance Criteria

- [x]  The shopping list screen (the screen that lists all of the user’s purchases) looks something like the following (or equivalent, use your creativity!):

## Description

* **Styled the the list view**: changed the header icon, the navigation links and the flash messages
* **Extra:** Added a new `UserToken.js` component that allows the user to copy the list token to clipboard 

## Related Issue

closes #15 
Closes #16 

## Updates

![Screen Shot 2020-11-27 at 6 30 43 PM](https://user-images.githubusercontent.com/36022831/100473764-611c0a80-30df-11eb-8817-d4cb5f5503f8.png)

## Testing Steps / QA Criteria

* Pull down this branch with git pull origin `hn-dg-issue-15` and `git checkout hn-dg-issue-15`
* Run `yarn start` to launch the app to see the shopping list page 🌟
* Join a list by using the **tina swamp rung** token, or create your own list and add some items to it.

Now you should see how the list and navbar look after the final style.

**Thank you :)**
